components/skus/browser/rs/wasm/src/lib.rs

```diff
@@ -177,20 +178,15 @@ impl JSSDK {
         let sdk = self.sdk.clone();
 
         let future = async move {
-            let window: ChromeWindow = web_sys::window()
-                .ok_or("couldn't get window")?
-                .dyn_into()
-                .unwrap();
+            let window: ChromeWindow =
+                web_sys::window().ok_or("couldn't get window")?.dyn_into().unwrap();
```

Comment:
just curious why we access to window.chrome.braveSkus from this lib?

Reply:
This is done so we do not use the wasm interface if we are running in a brave browser.  Effectively do not use the wasm wrapper in if we have brave skus built into the browser.